src/screens/OtherProfile/index.js

```diff
@@ -370,11 +370,13 @@ const OtherProfile = () => {
     setInitLoading(false);
   };
 
-  const fetchOtherProfile = async () => {
+  const fetchOtherProfile = async (preventCache = false) => {
     const status = await netInfo.fetch();
     if (status.isConnected) {
       try {
-        handleOfflineMode();
+        if (!preventCache) {
+          handleOfflineMode();
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fetchOtherProfile` function now accepts an optional parameter `preventCache`. If `preventCache` is true, the `handleOfflineMode` function will not be called. This seems like a logical change, but ensure that all calls to `fetchOtherProfile` have been updated accordingly.
```diff
- const fetchOtherProfile = async () => {
+ const fetchOtherProfile = async (preventCache = false) => {
+   if (!preventCache) {
+     handleOfflineMode();
+   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->